template/src/ExamplePanel.tsx

```diff
@@ -17,7 +23,7 @@ function ExamplePanel({ context }: { context: PanelExtensionContext }): JSX.Elem
     // Without a render handler your panel will never receive updates.
     //
     // The render handler could be invoked as often as 60hz during playback if fields are changing often.
-    context.onRender = (renderState: RenderState, done) => {
+    context.onRender = (renderState: Immutable<RenderState>, done) => {
```

Comment:
why do we include a type annotation for one parameter but not the other? Can we just omit both?

Reply:
Yeah we can probably leave it out. I'll double check and remove it once the studio release is out.